cmd/output/output.go

```diff
@@ -0,0 +1,66 @@
+package output
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+type (
+	Image struct {
+		Image  string `json:"image"`
+		Digest string `json:"digest"`
+	}
+	Data struct {
+		Registry    string  `json:"registry"`
```

Comment:
Use RegistryType instead of registry. Type needs to be enum for it https://stackoverflow.com/a/41480652
Similarly in annotation

Reply:
Enum is tricky to get marshaled, i reverted back to String enum like structure